classes/Device.class.php

```diff
@@ -2192,7 +2195,13 @@ static function UpdateSensors($CabinetID=null){
 			// Strip out everything but numbers
```

Comment:
I understand the added comma but why the rest when this comment right here says strip out everything but numbers.